pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -8,7 +8,7 @@ Date of Last Revision: November 17, 2016
 
 
 ## About Code.org
-Code.org® is a nonprofit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color.
+Code.org® is a nonprofit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented minorities.
```

Comment:
LGTM, we are allowed to make minor changes to the policy (as this is) without notifying our users. However, I would confirm the change with Hadi.